src/components/EventCard/EventCard.stories.tsx

```diff
@@ -6,27 +6,61 @@ export default {
   component: EventCard,
 };
 
+const exampleEventLiked = {
+  image:
+    "https://mobil.ksta.de/image/25684552/2x1/600/300/f6b5575094f8698991e7455e5a22460b/my/flohmarkt-symbolimagolem50254643h.jpg",
+  title: "Flohmarkt am Heumarkt",
+  startDate: "2021-12-12",
+  time: "11 bis 16 Uhr",
+  location: "Heumarkt",
+  fee: "Kostenlos",
+  link: "example",
+  endDate: "example",
+  description: "example",
+  zip: 123,
+  city: "example",
+  district: "example",
+  area: "example",
+  street: "example",
+  streetNumber: 123,
+  transportation: "example",
+  latitude: 123,
+  longitude: 123,
+};
+
+const exampleEventNotLiked = {
+  image:
+    "https://mobil.ksta.de/image/25684552/2x1/600/300/f6b5575094f8698991e7455e5a22460b/my/flohmarkt-symbolimagolem50254643h.jpg",
+  title: "Flohmarkt am Heumarkt",
+  startDate: "2021-12-12",
+  time: "11 bis 16 Uhr",
+  location: "Heumarkt",
+  fee: "Kostenlos",
+  link: "example",
+  endDate: "example",
+  description: "example",
+  zip: 123,
+  city: "example",
+  district: "example",
+  area: "example",
+  street: "example",
+  streetNumber: 123,
+  transportation: "example",
```

Comment:
You could consider other data than the string `example` to present a more "realistic" example in the storybook view.